services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/feature/BfdFeature.java

```diff
@@ -31,7 +31,7 @@
         if (description == null) {
             return empty;
         }
-        if (!MANUFACTURER_NOVIFLOW.equals(description.getManufacturerDescription())) {
+        if (!MANUFACTURER_DESCRIPTIONS_NOVIFLOW.contains(description.getManufacturerDescription())) {
```

Comment:
And where is NW440 regexp?

Reply:
Changed to regexp above "software description" field